app/helpers/application_helper.rb

```diff
@@ -370,10 +370,9 @@ def rails_asset_url_tag
   end
 
   def rails_asset_host_url
-    asset_host_enabled = Rails.configuration.asset_host.present?
     asset_host_url = Rails.configuration.three_scale.asset_host.presence
-
-    return '' unless asset_host_enabled && asset_host_url
+    return '' unless asset_host_url
+    return asset_host_url if asset_host_url.match? %r{^https?://}
 
     "#{request.protocol}#{asset_host_url}"
```

Comment:
In theory, nobody should access over http, but there are some hacks that customers can use to get http working https://access.redhat.com/solutions/3455941

I don't think we need to close that possibility completely.

Reply:
would such customer need to also access assets over `http` and at the same time specify a host without schema? I think that for customers insisting on http for assets, they can just specify a schema as well.

Reply:
@akostadinov I don't understand your question, but here's how it works:

- If `RAILS_ASSETS_HOST` specify a protocol, that one is used to generate assets urls.
- Otherwise, the same protocol being used by porta is used for assets.

Does that help?

Reply:
My point was that porta request schema doesn't have to be same as the assets schema. So it makes sense to hardcode https here like we do in porta configuration. And if one wants to use `http`, they can configure it explicitly.